localisation/00_wc_sad.csv

```diff
@@ -701,18 +640,15 @@ EVTDESC_WCCLS.10142;You have gained blessing of Aviana and will now face a final
 EVTOPTA_WCCLS.10142;I will not fail her.;;;;;;;;;;;;;x
 EVTDESC_WCCLS.10143;The ritual has begun. As the world swirls around you the vision blurs and you start to hear weird sounds. You see the sky swarming with foul demons. You nosedive into hordes of felbats sending many of them into the ground. You know you need to protect the skies and give coverage to your allies on the ground. You fight off countless flying demons while evading Infernals falling from above. Despite fighting with all your strength the demons ffinally overcome you. Your lifeless body is falling from the skies and the whole world swirls around you as you leave the vision.;;;;;;;;;;;;;x
 EVTOPTA_WCCLS.10143;I still weep for the Mistress of the Mother Tree.;;;;;;;;;;;;;x
-EVTDESC_WCCLS.10144;You have overcome the Avianas challenge and gained insight about her being. With all these things you have strengthened your connection with the Mother of Flight and mastered the Talon Form. You will use this power as Aviana herself would. To defend Azeroth and its inhabitants.;;;;;;;;;;;;;x
+EVTDESC_WCCLS.10144;You have overcome the Avianas challenge and gained insight about her being. With all these things you have strengthened your connection with the Mother of Flight and mastered the talon form. You will use this power as Aviana herself would. To defend Azeroth and its inhabitants.;;;;;;;;;;;;;x
 EVTOPTA_WCCLS.10144;I will always follow your example.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 study_at_temple;Study at the Temple;;;;;;;;;;;;;x
-study_at_temple_desc;Sisterhood of Elune has many powerful priestesses among its rank. You can ask them for training, increasing your class level in the process.;;;;;;;;;;;;;x
+study_at_temple_desc;The Sisterhood of Elune has many powerful priestesses among its ranks. You can ask them for training, increasing your class level in the process.;;;;;;;;;;;;;x
 study_at_temple_power_desc;Enables decision in the �YIntrigue�! menu, to study at the temple increasing your class level.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-has_remnant_of_caer_darrow;Has �YRemnant of Caer Darrow�! artifact;;;;;;;;;;;;;x
+has_remnant_of_caer_darrow;Has �YRemnant of Caer Darrow�! Artifact;;;;;;;;;;;;;x
 has_powerful_magical_artifact;Has a powerful magical artifact;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 EVTOPTD_WCSAD.140;I changed my mind.;;;;;;;;;;;;;x
-recently_turned_magi_tooltip;Has NOT turned someone into Two-headed ogre in the last 12 months.;;;;;;;;;;;;;x
+recently_turned_magi_tooltip;Has NOT turned someone into a two-headed ogre in the last 12 months.;;;;;;;;;;;;;x
```

Comment:
It should be Two-headed ogre

Reply:
If it refers to the trait, I would do this.
```suggestion
recently_turned_magi_tooltip;Has NOT turned someone into §YTwo-headed§! in the last 12 months.;;;;;;;;;;;;;x
```

Reply:
Ok, can do that

Reply:
done